.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ] Please Pull Request should have a descriptive title telling you what changes are inside linters are just a tool that the programmer use to make sure his code style is perfect but it is still not the main thing you did in this pull request creating linter.yml is not a descriptive title

- [ ] Please Pull Request should have a summary describing introduced changes in more detail
the main thing that you did in this pull request is adding an Html file and stlyle.css I don't see that merging the linter is a good summary for a pull request and please make sure to use actions to write this descriptive summary like `Add index.html` `Add a title say Hello Microverse` and so on


![image](https://user-images.githubusercontent.com/74744787/164070846-393a6e18-068d-46f9-950e-3305603ac654.png)

Reply:
cool, thanks!